tests/jest/components/AdminSettings.spec.js

```diff
@@ -969,31 +969,31 @@ describe('AdminSettings.vue', () => {
 					describe('upon failure', () => {
 						it.each([
 							[
-								'should set the project folder error message and error details when group folder app is not enabled',
+								'should set the project folder error message and error details when group folders app is not enabled',
 								{
-									error: 'The group folder app is not installed',
-									expectedErrorDetailsMessage: 'Please install the group folder to be able to use automatic managed folders or deactivate the automatically managed folders.',
+									error: 'The "Group folders" app is not installed',
+									expectedErrorDetailsMessage: 'Please install the "Group folders" app to be able to use automatic managed folders or deactivate the automatically managed folders.',
 								},
 							],
 							[
 								'should set the user already exists error message and error details when user already exists',
 								{
 									error: 'The user "OpenProject" already exists',
-									expectedErrorDetailsMessage: 'Please make sure to completely delete the previous user or deactivate the automatically managed folders.',
+									expectedErrorDetailsMessage: 'Setting up the OpenProject user, group and group folder was not possible. Please check this troubleshooting guide on how to resolve this situation.',
```

Comment:
The error message is the same so maybe we can add it in a variable somewhere and can reuse it?
```suggestion
									expectedErrorDetailsMessage: groupFolderSetupErrorMessage
```

Reply:
I think we can make a separate file containing all the error messages and use it in `vue` files and also in `specs`. This should not be in this PR so I'm resolving this for now. I'll create a separate work-packages